README.md

```diff
@@ -1,39 +1,90 @@
 
-```sh
-more-gray-shades
-support-dark-theme
-button-component
-itp.
-```
+#### Mentor:
 
-### Commit Message
+[Aleksander Atamańczuk](https://github.com/TenGosc007)
 
-Przyjmujemy poniższy format commit message
+#### Uczestnicy:
 
-```sh
-<type>: <commit_message>
-```
+- [Małgorzata Bednarczuk](https://github.com/margiebed) (Developer)
 
-#### Commit Message Type
+- [Anna Koruc](https://github.com/annakoruc) (UI)
 
-- **build**: Zmiana która afektuje budowanie projektu lub zależności zewnętrzne
-- **docs**: Zmiana w dokumentacji bądź dodatkowych materiałach
-- **feature**: Wprowadzenie nowej funkcjonalności
-- **bugfix**: Naprawienie błędu
-- **refactor**: Zmiana w projekcie, która nic nie naprawia, ani nie dodaje
+- [Marta Probierz](https://github.com/marta-probierz) (Product Manager)
 
-#### Zasady Commit Message
+- [Weronika Kurek-Pękala](https://github.com/SolWika) (Product Owner)
 
-- używaj imperatywnej formy czasu teraźniejszego: "change", nie "changed" lub "changes"
-- nie zaczynaj wielką literą
-- nie dodawaj kropki na końcu zdania
-- używaj języka angielskiego
+- [Kasia Filip](https://github.com/kasia-filip) (Developer)
 
-### Pull Request
+- [Sebastian Michalczyk](https://github.com/WindOfCodes) (Tech Lead)
 
-Pracujemy na gałęziach w metodologii `branch per feature`. Staramy się tworzyć małe PR. W opisie PR powinno być dokładnie opisane co on zmienia. Każdy PR musi być zaakceptowany przez przynajmniej jedną inną osobę.
+![Logo]()
+
+[Mockupy i prototyp](https://www.figma.com/file/j8rdQ9KEIYiAfFdH5DNgQM/schematy-PuppyLOG?node-id=0%3A1)
+   
+### [WEJDŹ NA STRONĘ](https://puppylog.netlify.app/)
+
+## Cel projektu
+Celem projektu było dostarczenie aplikacji "Puppylog" pozwalającej użytkownikom na odnalezienie odpowiedniego weterynarza dla swojego pupila. Zalogowany użytkownik może dokonać rezerwacji terminu wizyty, jak i samodzielnie dodawać swoje zwierzęta do bazy. Ponadto użytkownicy mogą otrzymywać powiadomienia o terminach szczepień lub, w przypadku dodania nowego zwierzęcia, zachęcenie do nich lub np. do chipowania. 
+
+Każde zwierzę posiada indywidualną kartę pacjenta. Widnieje w niej imię, gatunek, rasa, data urodzenia, daty szczepień, historia chorób oraz inne ważne informacje dotyczące pupila, np. numer chipa. Wszyscy weterynarze mają dostęp do karty danego zwierzęcia oraz mogą ją edytować. Dodatkowo karta pacjenta jest uzupełniana po każdej odbytej wizycie.
+
+Aplikacja została wykonana według wymagań dostarczonych przez organizatorów CodersCamp.
+
+
+## Działanie aplikacji
+
+#### Menu Główne
+
+...
+
+#### Lista Funkcjonalności
+
+...
+
+### Dodatkowe informacje
+
+Nasz zespół zrealizował także responsywność stron.
+
+## API
+
+...
+
+## Kod startowy projektu
+1. Nasza aplikacja została wdrożona na Netlify.
+2. Aplikacja jest budowana przy pomocy narzędzia Parcel.
```

Comment:
We don't use Parcel in this project, React use Webpack as default

Reply:
fixed